### DISCUSSION TITLE
Allow to change the Last Lap

### DISCUSSION THREAD

**Discussion by cael-gomes** [snipem/gt7dashboard#11] [Ideas]
Instead of always having the Last Lap as a fixed lap for comparison, allow users to select which laps they want to compare.
For instance, if the last lap is a cooldown lap, there is no way you can compare *another* lap of yours with the reference lap.

**Comment by Kalten70** [comment]
That is mentioned in the manual. In the Time Table at the bottom of the page, you can click individual lap rows and that data is added in the above charts. You can then turn them on/off from the chart, but not remove them. In the legend of each chart, you can turn on/off laps by clicking the lap, so you can turn off last lap if you want.
